specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/examples/ScriptRevisions_Get.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-567890abcdef",
+    "resourceGroupName": "SampleResourceGroup",
+    "workspaceName": "Workspace1",
+    "scriptName": "Script1",
+    "revision": 1,
+    "api-version": "2020-05-04-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 13 Sep 2017 18:04:32 GMT",
+        "x-ms-request-id": "d5496da4-9c52-402f-b067-83cc9ddea888",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-tenant-reads": "14999",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
+      },
+      "body": {
+        "kind": "SynapseSparkScript",
+        "properties": {
+          "scriptId": "a08f184b-0567-890a-ba22-a1199336d226",
+          "revision": 1,
+          "visibility": "ShowAll",
+          "sources": [
+            "foo"
+          ],
+          "sinks": [
+            "bar"
+          ],
+          "content": "Hello Python!",
+          "language": "Python"
+        },
+        "id": "/subscriptions/12345678-1234-1234-1234-567890abcdef/resourceGroups/SampleResourceGroup/providers/Microsoft.DataCollaboration/workspaces/Workspace1/scripts/Script1/revisions/1",
+        "name": "Script1",
```

Comment:
the name of the revision is "1" in the ID.  "Script1" is the name of the script.  The type should also be "Microsoft.DataCollaboration/workspaces/scripts/revisions"